src/components/select/Select.js

```diff
@@ -54,11 +54,17 @@ export default class SelectComponent extends BaseComponent {
     // Keep track of the select options.
     this.selectOptions = [];
 
-    // Keep track of the last batch of items loaded.
```

Comment:
So this will now work only if they are linked to our Resource? I don't think this is correct... this should work regardless if they are linked to our resources or not.